wizard_builder/migrations/0004_inheritence_downcasting.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-05-11 20:29
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wizard_builder', '0003_multisite'),
+    ]
+
+    operations = [
+        migrations.AlterModelManagers(
+            name='questionpage',
+            managers=[
+            ],
+        ),
+        migrations.AlterModelManagers(
+            name='textpage',
+            managers=[
+            ],
+        ),
+        migrations.RemoveField(
+            model_name='formquestion',
+            name='polymorphic_ctype',
+        ),
+        migrations.RemoveField(
+            model_name='pagebase',
+            name='polymorphic_ctype',
+        ),
+    ]
```

Comment:
id's definitely don't change, but I do definitely want to add a test for editing